index.html

```diff
@@ -19,14 +19,164 @@
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+        <ul class="workContainer">
+          <li id="workContainerhtml">HTML</li>
+          <li id="workContainercss">CSS</li>
+          <li id="workContainerjs">Javascript</li>
+        </ul>
+        <div>
+          <button type="button" class="cardButton">See Project</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</Section>
+<!-- __________________________________CARD4______________________________ -->
+<Section class="card-container">
+  <div class="card card1">
+    <img src="assets/CardPhoto4.svg" alt="ImgCard4">
+    <div class="cardWorks">
+        <h4>Tonic</h4>
+      <div>
+        <span class="word1">CANOPY</span>
+        <span class="dot"></span>
+        <span class="word2">Back End Dev</span>
+        <span class="dot"></span>
+        <span class="word3">2015</span>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+        <ul class="workContainer">
+          <li id="workContainerhtml">HTML</li>
+          <li id="workContainercss">CSS</li>
+          <li id="workContainerjs">Javascript</li>
+        </ul>
+        <div>
+          <button type="button" class="cardButton">See Project</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</Section>
+
+<!-- ______________________________About my self___________________________ -->
+<section class="aboutMyself">
+  <div class="aboutMyself-container">
+    <h3>About Myself</h3>
+    <p>Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don3t hesitate to contact me.
+    </p>
+        <h5>LET'S CONNECT</h5>
+        <div class="frame">
+        <a id="frame1" href=""></a>
+        <a id="frame2" href=""></a>
+        <a id="frame3" href=""></a>
+        <a id="frame4" href=""></a>
+        <a id="frame5" href=""></a>
+      </div>
+      <button type="button" id="aboutMyselfBotton">Get My Resume</button>
+  </div>
+  <div class="languageAndArrowContainer">
+    <span>Languages</span>
+    <img src="assets/downSideArrow.svg" alt="">
+  </div>
+  <div class="languagesContainer">
+    <ul>
+      <li class="languages"><img src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
+      <li class="languages"><img src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
+      <li class="languages"><img src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
+    </ul>
+  </div>
+  <div class="languageAndArrowContainer">
+    <span>Frameworks</span>
+    <img src="assets/leftSideArrow.svg" alt="leftSideArrow">
+  </div>
+  <hr>
+  <div class="languageAndArrowContainer">
+    <span>Skills</span>
+    <img class="skillsArrow" src="assets/leftSideArrow.svg" alt="leftSideArrow">
+  </div>
```

Comment:
Great work implementing most of the changes suggested by the previous reviewer. 👍🏿 
I would however suggest a tiny change regarding the position of the **drop-down-arrow**. 

As suggested by the previous reviewer, please make sure that the arrow is aligned close to the right side of the page and not in the center. 

- [ ] You can achieve this by simply changing the **flex justification** from `start` to `space-between` 🎯 

### Before suggestion ⬇️ 
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/117144028/291436c8-0399-470a-a3fa-717fee63078b)

### After suggestion
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/117144028/b9125be0-3202-4d9e-a423-8e1bb7d1444d)